server/__init__.py

```diff
@@ -5,7 +5,7 @@
 __version__ = "0.16.7"
 
 
-display_version = "cellxgene v" + __version__
+display_version = "CellxGene v" + __version__
```

Comment:
```suggestion
display_version = "CELLxGENE v" + __version__
```

Reply:
let's keep code unchanged 
```suggestion
display_version = "cellxgene v" + __version__
```